build on @reficul31's work a clean overhaul of the overview page.

blackforestboi opened the pull request:
the same changes as the unintended commit as a proper pull request. 

Overhaul of the overview, based on @reficul31's work. (#40)

## Screenshot
![screen shot 2017-04-02 at 02 44 33](https://cloud.githubusercontent.com/assets/7870039/24583569/0fa6dec6-174f-11e7-803c-e1d524c6bcc8.png)

blackforestboi committed:
build on @reficul31's work a clean overhaul of the overview page.

reficul31 commented:
Thanks @oliversauter . I thought the PR was merged. Although this code looks much more cleaner.

Treora commented:
It would have been cleaner to continue from @reficul31's branch, possibly after rebasing it onto master (you could even make a PR to his branch). Now it is hard to see what you wanted to change, and you reinsert some old stuff that was changed or removed.

The style changes themselves seem good; the code still needs a clean up, which was a reason why it was not merged yet.

blackforestboi committed:
1. removed uses of clustered css class
2. icons less opacity onhover of results
3. cleaned up css file for visit items (proper naming for classes)
4. a bit more structured css file (although that is only superficial to be able to read the blocks better)

blackforestboi committed:
file path from current master

blackforestboi commented:
@Treora 
Ok, next time will try to do it with rebase. 
I took the fork of reficul and compared the files with with the current master. 
I inserted the necessary divs and classes into the css files and removed things that are not needed anymore (i.e. caption class)
Also I renamed the existing classes to have more descriptive titles of what they actually do. 

> you reinsert some old stuff that was changed or removed.

I saw that I didn take the new file path `'src/page-viewer'` from the current master in file `visitListItem.js`. Any other things?

In the lastest commits I added a few other improvements, as seen in the commit message.

Here is the new screenshot. 

![screen shot 2017-04-02 at 12 08 16](https://cloud.githubusercontent.com/assets/7870039/24586312/a5ae77b4-179d-11e7-87ef-02fa1488f13a.png)

reficul31 commented:
@oliversauter @Treora Please tell me if I could assist in any way

blackforestboi committed:
Update Readme.md

blackforestboi committed:
Update Readme.md

Treora commented on Readme.md:
```diff
@@ -1,24 +1,20 @@
-# WebMemex/memextension
+# WorldBrain's WebMemex
```
So now you also want to put your project name in the WebMemex title? It's great you want to use and contribute to this project, but this may be a bit too much. ;)

blackforestboi replied:
ah meant to be temporary. I am also not satisfied with it completely. Want to give the WebMemex some sort of credit for it, didn't know how to do it. 

What do you think about "powered by WebMemex"?

blackforestboi replied:
I interpreted it as "WorldBrain's version of the WebMemex", but I see that WorldBrain's WebMemex sounds too much as if it is our work completely.

blackforestboi replied:
Oh, now I understand entirely! 
That title was not meant to be for your project. It was meant for the WorldBrain version of the WebMemex. 
Sorry for the confusion :)

Failed to exclude the Readme file from the PR to the Webmemex repo!

Treora replied:
Not sure you understood: I tried to point out that the changes in your WorldBrain fork ended up in your pull request to the WebMemex repo. It's often easiest to make a new branch for each pull request you make.

As for how you name your fork, that's another question.

Treora commented:
Continued in #55, issue resolved in de16361.
